js/admin.js

```diff
@@ -0,0 +1,31 @@
+jQuery( document ).ready(function($) {
+	$('#check-status').click(function(){
+		$(".theme-check-report").empty();
+		themeCheckRunPHPCS();
+	});
+	function themeCheckRunPHPCS(){
+		$.ajax({
+			url: ajaxurl,
+			type: 'POST',
+			data: {
+				'action': 'ns_theme_check_run',
+				'themename': $('select[name=themename]').val(),
+				'hide_warning': $('input[name=hide_warning]').is(':checked'),
+				'raw_output': $('input[name=raw_output]').is(':checked'),
+				'minimum_php_version': $('select[name=minimum_php_version]').val(),
+				'wordpress-theme': $('input[name=wordpress-theme]').is(':checked'),
+				'wordpress-core': $('input[name=wordpress-core]').is(':checked'),
+				'wordpress-extra': $('input[name=wordpress-extra]').is(':checked'),
+				'wordpress-docs': $('input[name=wordpress-docs]').is(':checked'),
+				'wordpress-vip': $('input[name=wordpress-vip]').is(':checked'),
+				'ns_theme_check_nonce': $('#ns_theme_check_nonce').val(),
+			},
+			success:function(data) {
+				$(".theme-check-report").html(data);
+			},
+			error: function(errorThrown){
+				console.log(errorThrown);
+			}
+		});
+	}
+});
```

Comment:
Ok, added the error logs, and again the same thing happens

~~~
[04-Apr-2017 19:20:33 UTC] /srv/www/wordpress-default/public_html/wp-content/plugins/ns-theme-check/vendor/bin/phpcs /srv/www/wordpress-default/public_html/wp-content/themes/flexible-lite --report-width=110 --report=json --standard=/srv/www/wordpress-default/public_html/wp-content/plugins/ns-theme-check/bin/phpcs.xml,WordPress-Theme --runtime-set default_standard WordPress-Theme --runtime-set ignore_warnings_on_exit 1 --runtime-set show_warnings 1 --ignore=.\*/node_modules/.\* --runtime-set testVersion 5.4-7.0 --runtime-set text_domain flexible-lite --runtime-set installed_paths /srv/www/wordpress-default/public_html/wp-content/plugins/ns-theme-check/vendor/wp-coding-standards/wpcs/
[04-Apr-2017 19:20:33 UTC] PHP Notice:  Undefined variable: raw_output in /srv/www/wordpress-default/public_html/wp-content/plugins/ns-theme-check/inc/checks.php on line 74
[04-Apr-2017 19:20:33 UTC] 
[04-Apr-2017 19:20:33 UTC] PHP Notice:  Undefined variable: return_var in /srv/www/wordpress-default/public_html/wp-content/plugins/ns-theme-check/inc/checks.php on line 75
[04-Apr-2017 19:20:33 UTC] 
~~~

But it works, so I guess this can be ignored?